Resources/Prototypes/_RMC14/Reagents/elements.yml

```diff
@@ -0,0 +1,499 @@
+- type: reagent
+  id: RMCAluminium
```

Comment:
idk why, but in cm-ss13 it called "Aluminum"
```
/datum/reagent/aluminum
name = "Aluminum"
```
from 'code/modules/reagents/chemistry_reagents/other.dm' line `461`